k8s_versions.json

```diff
@@ -1,6 +1,6 @@
 {
-  "v1.34.0": {
-    "digest": "sha256:7416a61b42b1662ca6ca89f02028ac133a309a2a30ba309614e8ec94d976dc5a",
-    "tag": "v1.34.0"
+  "v1.34.2": {
+    "digest": "sha256:745f8ed46d8e99517774768227fd1a0af34a6bf395aef9c7ed98fbce0e263918",
+    "tag": "v1.34.2"
```

Comment:
@frank-spano  is this expected ?
It is replacing the current latest `v1.34.x` version, I thought we should keep the `v1.34.0` and add the `v1.34.2` no ?